variable/reduction.cpp

```diff
@@ -205,15 +218,17 @@ void max_impl(const VariableView &out, const VariableConstView &var) {
 /// Variances are not considered when determining the maximum. If present, the
 /// variance of the maximum element is returned.
 Variable max(const VariableConstView &var, const Dim dim) {
-  return reduce_idempotent(var, dim, core::element::max_equals);
+  return reduce_idempotent(var, dim, core::element::max_equals,
+                           FillValue::Lowest);
 }
 
 /// Return the maximum along given dimension ignoring NaN values.
 ///
 /// Variances are not considered when determining the maximum. If present, the
 /// variance of the maximum element is returned.
 Variable nanmax(const VariableConstView &var, const Dim dim) {
-  return reduce_idempotent(var, dim, core::element::nanmax_equals);
+  return reduce_idempotent(var, dim, core::element::nanmax_equals,
+                           FillValue::Lowest);
```

Comment:
`FillValue::Lowest` used here, but equivalent for `min` below i` `FillValue::Max` -> should this be `FillValue::Min` rather than`FillValue::Lowest`

Reply:
Not sure I understand your question. `Min` would be close to 0, not `-max`.

Reply:
Indeed. Lowest is the correct solution.